formatTest/unit_tests/expected_output/bucklescript.re

```diff
@@ -4,60 +4,60 @@ bla#=(Some(10));
 
 bla#=someFunc(Some(10));
 
-test##var#=(Some(-10));
+test["var"] = Some(-10);
 
 obj##.prop;
 
 obj##.prod := exp;
 
-preview##style##border#=Js.string(
+(preview["style"]["border"] = Js.string)(
```

Comment:
this case is unfortunate, but it was present before. This change just makes it obvious.

with the currently released version of Reason:

```bash
echo 'preview##style##border#=Js.string("1px black dashed");' | refmt -p ml
;;(((preview ## style) ## border) #= Js.string) (("1px black dashed")
    [@reason.raw_literal "1px black dashed"])
```

Again, #2050 fixes the precedence for this case too, so I'm not worried.